### PR TITLE
Create permissions test suite with IDL tests

### PR DESCRIPTION
**Pull request by Honry**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/23605)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/335506252)

**Comment by Honry** [comment]
@jyasskin, @mounirlamouri, @marcoscaceres, I added you in the OWNERS file to help review Permissions API tests as you are spec editors, which means you'll get pinged on new PRs.  Please let me know if you want to be removed.

**Comment by mounirlamouri** [comment]
sgtm. @Honry did you look at the Blink tests? I wonder if there are some that could be moved to WPT.

**Comment by Honry** [comment]
> sgtm. @Honry did you look at the Blink tests? I wonder if there are some that could be moved to WPT.

Yeah, they are located at [here](https://chromium.googlesource.com/chromium/src.git/+/master/third_party/WebKit/LayoutTests/http/tests/permissions/), and I would like to move them to WPT.

**Comment by Honry** [comment]
@mounirlamouri, most of Blink tests require user interaction to invoke/revoke permissions, which is under implementing at #5671, so I will go back to it once the new WebDriver feature is online.

**Comment by lukebjerring** [comment]
What's happening with this PR?

**Comment by marcoscaceres** [comment]
I guess these could land to start testing the IDL? @Honry, is the IDL still up to date?

**Comment by Honry** [comment]
> @Honry, is the IDL still up to date?

@marcoscaceres, no, I will do it soon.

File: permissions/interfaces.any.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  } else {
+    idl_array.add_objects({ Navigator: ['navigator'] });
+  }
+
+  idl_array.add_objects({
+    Permissions: ['navigator.permissions'],
+    PermissionStatus: ['permissionStatus']
+  });
+  idl_array.test();
+  done();
```
Comment (marcoscaceres) [inline permissions/interfaces.any.js]: `done()` should be outside the test.

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
apart from `done()`, looks great @Honry!

**Comment by Honry** [comment]
@marcoscaceres, thanks for your review! `done()` is removed.

**Review by marcoscaceres — APPROVED** [review]
👌✨

**Pull request merged**